cmd/oras/internal/option/tty.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package option
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"golang.org/x/term"
+)
+
+const NoTTYFlag = "no-tty"
+
+// TTY option struct.
+type TTY struct {
+	noTTY bool
+	Tty   *os.File
```

Comment:
nit: Just wondering if this can be called `terminal`, so that we can avoid `Tty`😂

Reply:
Changed the struct name to `Terminal`.